bundles/Planning/Resources/assets/planning.js

```diff
@@ -0,0 +1,304 @@
+
+    const app = new Vue({
+        el: '#app',
+        data: {
+            hideUsers: {},
+            toggle: {},
+            sprints: [],
+            users: {},
+            projects: {},
+            numberLoaded: 0,
+            projectFilter: '',
+            userFilter: ''
+        },
+        computed: {
+            sortedUsers: function () {
+                if (this.users === {}) {
+                    return [];
+                }
+
+                var arr = Object.keys(this.users).map(function (i) {
+                    return this.users[i];
+                }.bind(this));
+
+                arr = arr.filter(function (item) {
+                    return item.displayName.toLowerCase()
+                        .indexOf(this.userFilter) !== -1;
+                }.bind(this));
+
+                arr = arr.sort(function (a, b) {
+                    if (a.key === 'unassigned') {
+                        return 1;
+                    }
+                    return (a.displayName.toLocaleLowerCase() > b.displayName.toLocaleLowerCase()) ? 1 : -1;
+                });
+
+                return arr;
+            },
+            sortedProjects: function () {
+                if (this.projects === {}) {
+                    return [];
+                }
+
+                var arr = Object.keys(this.projects).map(function (i) {
+                    return this.projects[i];
+                }.bind(this));
+
+                arr = arr.filter(function (item) {
+                    return item.name.toLowerCase()
+                        .indexOf(this.projectFilter) !== -1;
+                }.bind(this));
+
+                arr = arr.sort(function (a, b) {
+                    return (a.name.toLocaleLowerCase() > b.name.toLocaleLowerCase()) ? 1 : -1;
+                });
+
+                return arr;
+            }
+        },
+        created: function () {
+            var hideUsers = localStorage.getItem('hideUsers');
+
+            if (hideUsers !== null) {
+                this.hideUsers = JSON.parse(hideUsers);
+            }
+
+            axios.get(currentUrl + '/future_sprints')
+                .then(function (response) {
+                    this.sprints = response.data.sprints;
+
+                    for (var i = 0; i < this.sprints.length; i++) {
+                        this.getSprint(this.sprints[i].id, i);
+                    }
+                }.bind(this))
+                .catch(function (error) {
+                    console.log(error);
+                });
+        },
+        methods: {
+            toggleUser: function (key) {
+                var newValue = !this.hideUsers[key];
+                Vue.set(this.hideUsers, key, newValue);
+                localStorage.setItem('hideUsers', JSON.stringify(this.hideUsers));
+            },
+            getToggle: function (key) {
+                var toggled = this.toggle.hasOwnProperty(key) && this.toggle[key];
+
+                if (toggled) {
```

Comment:
`return toggled ? '<i class="fas fa-angle-up"></i>' : '<i class="fas fa-angle-down"></i>';`, please!

Or bettet yet (IMHO) use a css class on the parent element for this.